--- /home/premanandal1978/android16-i001d-analysis/unpacked/android14-files/vendor/etc/init/hw/init.target.rc	2009-01-01 00:00:00.000000000 +0000
+++ /home/premanandal1978/android16-i001d-analysis/unpacked/android16-files/vendor/etc/init/hw/init.target.rc	2009-01-01 00:00:00.000000000 +0000
@@ -63,8 +63,6 @@
     start vendor.sec_nvm
 
 on late-fs
-    wait_for_prop hwservicemanager.ready true
-    exec_start wait_for_keymaster
     mount_all /vendor/etc/fstab.qcom --late
 
 on post-fs-data
@@ -155,7 +153,6 @@
 on property:init.svc.vendor.charger=running
     start vendor.power_off_alarm
     setprop sys.usb.controller a600000.dwc3
-    setprop sys.usb.configfs 1
 
 service vendor.spdaemon /vendor/bin/spdaemon
     class core
