diff --git a/BoardConfig.mk b/BoardConfig.mk
index 5ee2ef1..51c64f6 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -9,6 +9,7 @@
 DEVICE_PATH := device/asus/I001D
 
 BUILD_BROKEN_CLANG_PROPERTY := true
+BUILD_BROKEN_MISSING_REQUIRED_MODULES := true
 
 # Assertions
 TARGET_OTA_ASSERT_DEVICE := WW_I001D,I001D,ZS660KL
diff --git a/bliss_I001D.mk b/bliss_I001D.mk
index d83827e..4ae4350 100644
--- a/bliss_I001D.mk
+++ b/bliss_I001D.mk
@@ -17,6 +17,9 @@ $(call inherit-product, device/asus/I001D/device.mk)
 # Bootanimation
 TARGET_BOOT_ANIMATION_RES := 1080
 
+BUILD_BROKEN_MISSING_REQUIRED_MODULES := true
+TARGET_FACE_UNLOCK_SUPPORTED := false
+
 # UDFPS Animations
 EXTRA_UDFPS_ANIMATIONS := true
 
@@ -36,3 +39,4 @@ PRODUCT_BUILD_PROP_OVERRIDES += \
     TARGET_DEVICE=ZS660KL \
     PRODUCT_NAME=WW_I001D
     BUILD_FINGERPRINT := asus/WW_I001D/ASUS_I001_1:11/RKQ1.200710.002/18.0210.2201.215-0:user/release-keys
+BUILD_BROKEN_MISSING_REQUIRED_MODULES := true
